### ISSUE TITLE
better description for 'min_structure_distance'

### ISSUE THREAD

**Issue by EnderMelody** [Silverminer007/Shrines#47]
im trying to make structures rarer, but i cannot tell what 'min_structure_distance' is supposed to do based on the description, and changing the value makes no structures generate on both ends (1 and 100). can you explain/ improve the description for what this is supposed to do.

**Comment by Silverminer007** [comment]
It's a little tricky with this one. Min structure distance defines a amount of chunks within no other structure is allowed to generate.
This means, when you set it to 100, this makes all structures super rare (in theory), because a structure can only spawn if there is no other structure *possible spawn* in a circle with a radius of 100 chunks = 1600 blocks.
This worked in the first versions of the mod for older minecraft versions, as there often were like 5 structure of this mod on one spot. But in more recent versions, Minecraft changed the way structures work and there are neary everywhere *possible* structure spawns. Therefore, I have removed this option in the lastest update (Mc1.20.1)

TLDR: The option doesn't really work on anything higher than 1.16.3 really, set it to 0 and use one of the options below:

There are different options to tweak the distance between structures based on the version you're playing:
- 1.20 & 1.19 & 1.18: Shrines Structures are based on datapacks. You should download either the datapack directly or the "Shrines default config" from github releases, extract that, change the files you want and re-zip the datapack ... (I know you need more knowledge for this, but this is how it works)
- 1.16.5 -> use the GUI which opens when you press "k" when you're in a world. Maybe this also works for 1.18, but I can't remember
